packages/vouching/contracts/Vouching.sol

```diff
@@ -0,0 +1,97 @@
+pragma solidity ^0.4.24;
+
+
+import "./ZepToken.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+contract Vouching {
+  using SafeMath for uint256;
+
+  struct Dependency {
+    address owner;
+    address dependencyAddress;
+    uint256 stake;
+  }
+
+  mapping (string => Dependency) private _registry;
+  uint256 private _minimumStake;
+  ZepToken private _token;
+
+  modifier onlyDependencyOwner(string name) {
+    require(msg.sender == _registry[name].owner, "Not allowed");
+    _;
+  }
+
+  event OwnershipTransferred(address indexed oldOwner, address indexed newOwner);
+  event DependencyCreated(string name, address indexed owner, address indexed dependencyAddress, uint256 initialStake);
+  event Vouched(string name, uint256 amount);
+  event Unvouched(string name, uint256 amount);
+  event DependencyRemoved(string name);
```

Comment:
Nit, we usually place the events right under the contract name

Reply:
:+1: good to know. Updated accordingly.